lib/node_modules/@stdlib/blas/base/drot/test/test.drot.js

```diff
@@ -0,0 +1,576 @@
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Float64Array = require( '@stdlib/array/float64' );
+var dcopy = require( '@stdlib/blas/base/dcopy' );
+var EPS = require( '@stdlib/constants/float64/eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var drot = require( './../lib/drot.js' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof drot, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function has an arity of 7', function test( t ) {
+	t.strictEqual( drot.length, 7, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function applies a plane rotation with x stride as 1 and y stride as 1', function test( t ) {
+	var xexpected;
+	var yexpected;
+	var nvalues;
+	var xvalues;
+	var yvalues;
+	var delta;
+	var tol;
+	var out;
+	var ex;
+	var ey;
+	var ix;
+	var iy;
+	var c;
+	var j;
+	var k;
+	var x;
+	var y;
+
+	xexpected = [
+		[ 0.6, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, -0.46, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, -0.46, -0.22, 1.06, 0.9, -0.3, -0.4 ]
+	];
+	yexpected = [
+		[ 0.5, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.04, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.04, -0.78, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.04, -0.78, 0.54, 0.08, -0.6, 0.2, 0.8 ]
+	];
+	nvalues = [ 0, 1, 2, 4 ];
+	ix = 1;
+	iy = 1;
+
+	xvalues = [
+		[ 0.6, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ]
+	];
+	yvalues = [
+		[ 0.5, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ]
+	];
+
+	c = 0;
+	for ( j = 0; j < nvalues.length; j++ ) {
+		x = new Float64Array( xvalues[ 0 ] );
+		y = new Float64Array( yvalues[ 0 ] );
+		out = drot( nvalues[ j ], x, ix, y, iy, 0.8, 0.6 );
+		ex = new Float64Array( xexpected[ c ] );
+		ey = new Float64Array( yexpected[ c ] );
+		c += 1;
+		for ( k = 0; k < ex.length; k++ ) {
+			if ( x[ k ] === ex[ k ] ) {
+				t.strictEqual( x[ k ], ex[ k ], 'returns expected value' );
+			} else {
+				delta = abs( x[ k ] - ex[ k ] );
+				tol = 4.0 * EPS * abs( ex[ k ] );
+				t.ok( delta <= tol, 'within tolerance. x: '+x[ k ]+'. expected: '+ex[ k ]+'. delta: '+delta+'. tol: '+tol+'.' );
+			}
+			if ( y[ k ] === ey[ k ] ) {
+				t.strictEqual( y[ k ], ey[ k ], 'returns expected value' );
+			} else {
+				delta = abs( y[ k ] - ey[ k ] );
+				tol = 4.0 * EPS * abs( ey[ k ] );
+				t.ok( delta <= tol, 'within tolerance. y: '+y[ k ]+'. expected: '+ey[ k ]+'. delta: '+delta+'. tol: '+tol+'.' );
+			}
+			if ( out[ k ] === y[ k ] ) {
+				t.strictEqual( out[ k ], y[ k ], 'returns expected value' );
+			} else {
+				delta = abs( out[ k ] - y[ k ] );
+				tol = 4.0 * EPS * abs( y[ k ] );
+				t.ok( delta <= tol, 'within tolerance. out: '+out[ k ]+'. expected: '+y[ k ]+'. delta: '+delta+'. tol: '+tol+'.' );
+			}
```

Comment:
This is not necessary. Above, just do

```javascript
t.strictEqual( out, y, 'returns expected value' );
```

Meaning, assert that the return value is the same reference as `y` and your `y` tests above cover both `out` and `y`.